projects/vdk-plugins/vdk-meta-jobs/tests/test_meta_job.py

```diff
@@ -327,6 +332,11 @@ def test_meta_job_duplicate_jobs(self):
     def test_meta_job_not_allowed_job_key(self):
         self._test_meta_job_validation("meta-job-not-allowed-job-key")
 
+    def test_meta_job_wrong_job_arguments_json(self):
```

Comment:
wher is the happy path tested ? That you successfully passed arguments to the job ?

You should test a few happy paths : 
* Where arguments are missing - None (which is already tested by previous test
* Where arguments is empty {} 
* Norma {"key: value" , "key2": value}

Reply:
Added them but forgot an log assertion that the arguments are there. Will address this and push later.

Reply:
Added the assertions.